src/UnitTests/Analysis/ValuePropagationTests.cs

```diff
@@ -471,7 +472,43 @@ public void VpPhiWithConstants()
             Assert.AreEqual("r3 = 0x4711", instr.ToString());
         }
 
-		private class DpbMock : ProcedureBuilder
+        [Test(Description = "(x = phi(<exp>, <exp>, x, x)) ==> (x = <exp>)")]
+        public void VpPhiLoops()
```

Comment:
Isn't the description a little misleading? What you're doing here is making sure that if `x = phi(a_1, a_2, .. a_n)` and all the `a_n`, after value propagation, are equal, then replace the original expression with `x = a_n`.